intents docs migration (#2458)
near/docs: docs/2.build/1.chain-abstraction/what-is.md

NEAR's chain abstraction framework consists of three core technologies that work together to create seamless cross-chain experiences:

1. [**NEAR Intents**](#NEAR-Intents): A decentralized system where users express desired outcomes (like "swap Token A for Token B at the best price") without specifying technical details. A network of solvers then competes to fulfill these intents optimally, handling complex cross-chain operations behind the scenes.

2. [**Chain Signatures**](#chain-signatures): Enables NEAR accounts, including smart contracts, to sign and execute transactions on other blockchains (like Bitcoin or Ethereum), allowing cross-chain interactions.

3. [**OmniBridge**](#omnibridge): A trustless multi-chain asset bridge that combines Chain Signatures for cross-chain transaction execution with a verification layer allowing NEAR smart contracts to confirm transactions on foreign chains. This creates a fully trustless system where NEAR can both initiate and verify cross-chain operations.

### NEAR Intents

> The NEAR Intents is a new type of transaction that allows information, requests, assets, and actions to be exchanged between users, services, and AI agents.

Author: ```suggestion
The [NEAR Intents](../../1.concepts/intents/overview.md) is a new type of transaction that allows information, requests, assets, and actions to be exchanged between users, services, and AI agents.
```

> The [NEAR Intents](../../chain-abstraction/intents/overview.md) is a new type of transaction that allows information, requests, assets, and actions to be exchanged between users, services, and AI agents.

Author: ```suggestion
[NEAR Intents](../../chain-abstraction/intents/overview.md) are a new type of transaction that allows information, requests, assets, and actions to be exchanged between users, services, and AI agents.
```
